pgmpy/tests/test_models/test_FunctionalBayesianNetwork.py

```diff
@@ -162,6 +170,146 @@ def test_simulate_different_distributions(self):
         self.assertTrue(np.all(samples["lognormal"] > 0))
         self.assertTrue(np.all(samples["gamma"] > 0))
 
+    def test_svi_fit_normal(self):
+        alpha = 0.23
+        x1 = np.random.normal(1, 2, size=5000)
```

Comment:
I would suggest reducing the sample size to `1000` to make it faster. Else, it would slow down our github actions.